resources/views/single-incident.blade.php

```diff
@@ -8,21 +8,22 @@
 
 @section('content')
 <h1>{{ formatted_date($incident->created_at) }}</h1>
-
+@foreach($incident_history as $incidentHistory)
```

Comment:
Ah, instead of showing a single incident you ALWAYS loop through the IncidentHistory of the given Incident. Good idea!